File: backend2/BusinessService/Common/Bases/Response/CmDownloadResponseBase.cs
// *******************************************************************
// 業務名称　: 互助防疫システム
// 機能概要　: ダウンロード処理
// 　　　　　　レスポンスインターフェースベース
// 作成日　　: 2024.07.14
// 作成者　　: 蔣
// 変更履歴　:
// *******************************************************************
namespace BCC.Affect.Service
{
    public class CmDownloadResponseBase : CmPreviewResponseBase
    {
        public EnumDownloadType downloadtype { get; set; } = EnumDownloadType.Data; //ファイルタイプ
        public string filefullnm { get; set; }                                      //ファイル名(パス含む)
    }
}

File: backend/JbdGjsService/Common/Bases/Response/CmDownloadResponseBase.vb
﻿' *******************************************************************
' 業務名称　: 互助事業システム
' 機能概要　: ダウンロード処理
' 　　　　　　レスポンスインターフェースベース
' 作成日　　: 2024.07.21
' 作成者　　: AIPlus
' 変更履歴　:
' *******************************************************************
Namespace JBD.GJS.Service
    Public Class CmDownloadResponseBase
        Inherits CmPreviewResponseBase
        Public Property downloadtype As EnumDownloadType = EnumDownloadType.Data 'ファイルタイプ
        Public Property filefullnm As String                                      'ファイル名(パス含む)

        Public Sub New()

        End Sub

        Public Sub New(msg As String)
            MyBase.New(msg)
        End Sub

        Public Sub New(recode As EnumServiceResult,msg As String)
            MyBase.New(recode, msg)
        End Sub

    End Class
End Namespace
